resources/windows/wix/directorylist.wxs.erb

```diff
@@ -2,8 +2,28 @@
 <Wix xmlns='http://schemas.microsoft.com/wix/2006/wi' xmlns:util="http://schemas.microsoft.com/wix/UtilExtension">
 
   <Fragment>
+
+    <!-- Service Directories derived from service definitions.-->
     <DirectoryRef Id='TARGETDIR'>
-      <%= @platform.generate_wix_dirs(self.get_services) %>
+      <Directory Id="<%= settings[:base_dir] %>" >
+        <Directory Id="<%= settings[:company_id] %>" Name="<%= settings[:company_name] %>">
+          <Directory Id='INSTALLDIR' Name="<%= settings[:product_id] %>" />
+        </Directory>
+      </Directory>
+      <Directory Id="CommonAppDataFolder" Name="CommonAppData">
+        <Directory Id="<%= settings[:company_id] %>AppData" Name="<%= settings[:company_id] %>">
+        </Directory>
+      </Directory>
+    </DirectoryRef>
+
+    <DirectoryRef Id="INSTALLDIR">
+      <Directory Name="puppet" Id="puppet">
+        <Directory Name="bin" Id="bin">
+          <Directory Id="PUPPETBINDIR" />
+          <Directory Id="MARIONETTECOLLECTIVEBINDIR" />
+          <Directory Id="PXPAGENTBINDIR" />
+        </Directory>
+      </Directory>
```

Comment:
For anyone looking this over, this is temporary work that @jcoconnor did to get his own stuff to work while I refactor the directorylist vanagon work to reflect some changes. This will change at some point